packages/@ama-sdk/generator-sdk/jest.config.js

```diff
@@ -15,6 +15,10 @@ module.exports = {
     ['jest-junit', {outputDirectory: resolve(__dirname, 'dist-test'), outputName: 'ut-report.xml'}],
     'github-actions'
   ],
+  fakeTimers: {
+    // TODO enable fake timers
+    // enableGlobally: true
```

Comment:
Why it is not enabled in this PR?

Reply:
it didn't seem that easy to make it work with `yeoman-test`